modules/dailymotionBidAdapter.js

```diff
@@ -90,6 +92,20 @@ function getVideoMetadata(bidRequest, bidderRequest) {
   return videoMetadata;
 }
 
+/**
+ * Check if user sync is enabled for Dailymotion
+ *
+ * @return boolean True if user sync is enabled
+ */
+function isUserSyncEnabled() {
+  const syncEnabled = deepAccess(config.getConfig('userSync'), 'syncEnabled');
+
+  const canSyncWithIframe = syncEnabled && userSync.canBidderRegisterSync('iframe', 'dailymotion');
+  const canSyncWithPixel = syncEnabled && userSync.canBidderRegisterSync('image', 'dailymotion');
+
+  return !!(canSyncWithIframe || canSyncWithPixel);
```

Comment:
Not sure the `!!` cast is necessary here because the `can...` variables already contain booleans

Reply:
It's just to ensure type safety, we never know 🤷